src/main/java/frc/robot/Robot.java

```diff
@@ -200,6 +200,9 @@ public void robotSetup() {
     this.climberSubsystem.resetClimberPosition();
     ahrs = new AHRS(SPI.Port.kMXP); 
 
+    HIDTester driverHIDTester = new HIDTester(RobotMap.DRIVER_PAD_PORT);
```

Comment:
You might end up with a port conflict having both a `HIDTester` and an `XBoxOneController` on the same port. I would try keeping the two existing Dual Actions on USB 0 and 1, and then add an XBoxOneController on USB 2 and verify the controls mapping that way, then make the switchover.

Reply:
Okay makes sense, thank you